websites/P/Picarto/dist/metadata.json

```diff
@@ -1,20 +1,19 @@
 {
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "Picarto",
   "author": {
     "name": "TheSlowly",
     "id": "505071508851523599"
   },
-  "service": "Picarto",
+  "category": "videos",
+  "logo": "https://i.imgur.com/Oljre0e.png",
+  "thumbnail": "https://i.imgur.com/DSNnMk5.png",
+  "version": "1.2.8",
+  "color": "#ff0000",
+  "tags": ["video"],
+  "url": "picarto.tv",
   "description": {
     "en": "Picarto is the world’s leading live streaming platform for creative minds. Come join us and watch artists from all over the world during their creative process!",
     "nl": "Picarto is 's werelds toonaangevende live streamingplatform voor creatieve geesten. Doe mee en kijk naar artiesten van over de hele wereld tijdens hun creatieve proces!"
-  },
-  "url": "picarto.tv",
-  "version": "1.2.7",
-  "logo": "https://i.imgur.com/Oljre0e.png",
-  "thumbnail": "https://i.imgur.com/DSNnMk5.png",
-  "color": "#FF0000",
-  "tags": [
-    "video"
-  ],
-  "category": "videos"
-}
\ No newline at end of file
+  }
+}
```

Comment:
there is no need to change the order...

Reply:
There kinda is, it broke for me without the old metadata?

Reply:
bruh it's json... it doesn't matter.
You still just do metadata.author etc.